README.md

```diff
@@ -306,6 +306,25 @@ It's also possible to customize the `Allow-Origin` but setting an environment va
 ENV NGINX_CORS_ALLOW_ORIGIN="https://my-domain.cool"
 ```
 
+## For Nginx <-> PHP communication customization
+
+By default Nginx and PHP communicate over a Unix socket for greater performance. But there are situations where you 
```

Comment:
I think we can remove our opinion from here, the documentation is like "how to use tcp instead of unix socket for fpm"